pkg/operator/operator.go

```diff
@@ -482,3 +485,19 @@ func removeFinalizer(instance *v1alpha1.Driver, f string) {
 	}
 	instance.ObjectMeta.Finalizers = result
 }
+
+func (c *csiDriverOperator) recorderForInstance(instance *v1alpha1.Driver) events.Recorder {
+	// Guess Kind run-time, just in case it's changed in types.go
+	t := reflect.TypeOf(instance)
+	kind := t.Elem().Name()
```

Comment:
`resourcehelper.GuessObjectGroupVersionKind(instance)`?

Reply:
Tried that, got `<unknown>` Kind.